pyproject.toml

```diff
@@ -60,6 +60,7 @@ autoflake = "^1.4"
 flake8 = "^3.9.1"
 aiohttp = "^3.7.4"
 aiohttp_cors = "^0.7.0"
+jupyter = "^1.0.0"
```

Comment:
No we shouldn't depend on Jupyter for dev. We already depend on `ipykernel`. You only need 1 jupyter for the whole machine and not for individual packages

Reply:
This is removed. I had to add a dev dependency on the widgets though or we wouldn't be able to use tqdm's html progress bar and it can't pull that package from the system environment it seems.